src/rules/require-each-key.ts

```diff
@@ -0,0 +1,27 @@
+import type { AST } from "svelte-eslint-parser"
+import { createRule } from "../utils"
+
+export default createRule("require-each-key", {
+  meta: {
+    docs: {
+      description: "require keyed `{#each}` block",
+      category: "Best Practices",
+      recommended: false,
```

Comment:
I think we can toggle to true in major version up?

Reply:
Hmm. I personally like to enforce the key, but `vue/require-v-for-key` is unpopular even in eslint-plugin-vue, so I think it's better not to include it in the preset. (However, the Vue style guide recommends using keys.)
I think it will be easier to introduce eslint if the presets are not too strict.

Reply:
Okay!